azure-pipelines-release.yml

```diff
@@ -0,0 +1,135 @@
+trigger: none
+pr: none
+
+pool:
+  name: '1ES-Hosted-DurableTaskFramework'
+  demands:
+    - ImageOverride -equals MMSWindows2022Custom
+
+steps:
+# Use NuGet
+- task: NuGetToolInstaller@1
+  displayName: 'Use NuGet '
+
+# dotnet restore
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet restore'
+  inputs:
+    command: restore
+    projects: '**/**/*.csproj'
+    feedsToUse: config
+    nugetConfigPath: '.nuget/nuget.config'
+
+# Build durable-extension
+- task: VSBuild@1
+  displayName: 'Build Durable Extension'
+  inputs:
+    solution: '**/WebJobs.Extensions.DurableTaskNoTest.sln'
```

Comment:
Why did we add a solution file here? We should just use the existing solution file.